migration/v0.34/apply/30-delete-obsolete-ns.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+run() {
+  case "${1:-}" in
+  execute)
+    # Did it exist in one or both namespaces?
+    for CLUSTER in sc wc; do
+      for NS in elastic-system influxdb-prometheus; do
+        if kubectl_do $CLUSTER get namespace $NS >/dev/null 2>/dev/null; then
+          # Sensible to use warn for potentially destructive actions?
```

Comment:
You should probably remove the comments at line 11 and 15.
They seem more like questions for people, then comments meant to show how a function in the code works.

Reply:
Done.